doc/guides/modules.rst

```diff
@@ -44,6 +44,9 @@ Module Repositories
 * The module repository codebase shall include a *module.yml* file in a
   :file:`zephyr/` folder at the root of the repository.
 
+* The *module.yml* file shall contain the ``name`` key and have the value
+  identical to the ``project`` name in the default manifest file.
```

Comment:
You need to rephrase that a bit...

```and shall have an identical value to the....``` 

maybe?

Reply:
Good point. Will wait with an update, until we have made a final conclusion on this topic.